apps/namada-interface/.prettierrc

```diff
@@ -2,5 +2,6 @@
   "trailingComma": "es5",
   "tabWidth": 2,
   "semi": true,
-  "singleQuote": false
+  "singleQuote": false,
+  "bracketSpacing": false
```

Comment:
I noticed `"bracketSpacing": false` also got set in `apps/extension/.prettierc` in one commit but removed again in another commit. Is it supposed to be added here?

Reply:
I should probably remove this as well. It fixed annoying formatting issues in one case, but introduced another.